Algebraic Topology/algebraic-topology.tex

```diff
@@ -454,9 +454,94 @@ \section{Functoriality of the fundamental group}
 \end{proof}
 
 
+\mydate{d7}{8}{2}{2017}
 
+\section{CW complexes and the fundamental group}
 
+Consider the polygon
 
+\begin{figure}
+  % \includegraphics{}
+  \label{fig:star}
+\end{figure}
+
+$X_0$ = picture
+
+$X_1$ = picture ((it seems I am inconsistent with whether these are capital x's or lowercase.  Should they all be capital?  I went back and made them all capital.))
+
+\begin{thm}
+  If $X$ is a CW complex and $A$ a contractible subcomplex, then $X$ is \textbf{not} equal to $X/A$ (this is the quotient by the equivalence relation $x \sim x'$ iff $x = x'$ or $x, x' \in A$).
+\end{thm}
+
+Once we hit $X_1$, the connected components of the complex are determined.
+
+That's because for any $n$-dimensional disk where $n \geq 2$, it's boundary is connected, so that boundary must lie entirely in \textbf{one} of the connected components.
+
+In $\ref{fig:star}$, the set of connected components is $\pi_0(X_0) = \pi_0(X_1)$ ((let's make the fund command take in an optional argument, which will be the subscript of the pi))
+
+If $X_0 = \{[*]\}$,
+
+$\pi_1(x_1) \isom F_{\text{set of 1-cells}}$
+
+$\pi\left( \frac{X_1 \union D^2}{\sim} \right) \isom \frac{\fund(B) * \fund(A)}{A \cap B} \isom \frac{\fund(x_1)}{[\phi] = 1}$, where * denotes the free group product. (( i think he meant $\fund$ instead of $\pi$ at the beginning))
+
+In general, I have many gluing maps $\phi_\alpha\from S^1 \to X_1$ and I glue $D^2\alpha$ along $\phi_\alpha(S^1)$.  So $\fund(X_2) \isom \fund(X_1)/\langle[\phi_\alpha]\rangle$.  ((this one makes my head spin.  i dont get it))
+
+\begin{thm}[Not to be proven]
+  Every group is the fundamental group of some CW complex.  Construct as follows:
+
+  $X_0$ = point
+
+  $X_1$ = every element in the group gets a loop (the free group)
+
+  $X_2$ = every entry in the multiplication table gets a disk (a relation)
+
+  So it's just like a group presentation!
+\end{thm}
+\begin{thm}
+  Every finitely presented group is the fundamental group of a finite CW complex.
+\end{thm}
+\begin{thm}
+  If $n > 3$, then for any finitely presented group $G$, there exists a compact manifold $M$ s.t. $G \isom \fund(M)$.
+\end{thm}
+\begin{proof}
+  If $n > 2$ and $M$ and $N$ are $n$-manifolds, then $M \# N$
+
+  % \includegraphics{} ((that puncture-glueing thing))
+
+  $F_n \isom \fund(\#^n(S^{n-1} \x S^1)) = \fund(M \# N) \isom \fund(M) * \fund(N)$
+
+  and $\delta(S^0 \x D^n) = S^0 \x S^{n-1} - \delta(D^1 \x S^{n-1})$ ((ermm, maybe he means set subtraction)) ((what does it all mean.  surely there is an error in this line.  is delta meant to denote boundary?))
```

Comment:
The idea is relatively simple if you work it out for n-1. If you do n=2, it should be trivially true.